packages/neuron-wallet/src/block-sync-renderer/sync/indexer-cache-service.ts

```diff
@@ -153,17 +151,36 @@ export default class IndexerCacheService {
     return mappingsByTxHash
   }
 
+  private async getCacheBlockNumber() {
+    if (!this.#cacheBlockNumberEntity) {
+      let lastCacheBlockNumber = await getConnection().getRepository(SyncInfoEntity).findOne({ name: `${SyncInfoEntity.LAST_CACHE_BLOCK_NUMBER}_${this.walletId}` })
+
+      if (!lastCacheBlockNumber) {
+        lastCacheBlockNumber = new SyncInfoEntity()
+        lastCacheBlockNumber.name = `${SyncInfoEntity.LAST_CACHE_BLOCK_NUMBER}_${this.walletId}`
```

Comment:
A method of getting the key can be defined in `SyncInfoEntity` so it won't be constructed repeatedly.

And the initialization can be defined in `SyncInfoEntity` too.